.github/workflows/production-heartbeat.yml

```diff
@@ -0,0 +1,103 @@
+name: production-heartbeat
+on:
+  schedule:
+    # Times are UTC
+    - cron: '45 1,5,9,13,17,21 * * *'
+jobs:
+  production-heartbeat:
+    strategy:
+      # By default, if any job in a matrix fails, all other jobs are immediately cancelled. This makes the jobs run to completion instead.
+      fail-fast: false
+      matrix:
+        os: [{vm: ubuntu-latest, exe: .sh}, {vm: windows-2019, exe: .cmd}]
+        node: ['lts/*']
+    runs-on: ${{ matrix.os.vm }}
+    steps:
+      # === Setup. We need to get the code, set up nodejs, and create the results directory. ===
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: mkdir smoke-test-results
+      # === Make three attempts to install sfdx through npm ===
+      - name: SFDX install first try
+        id: sfdx_first_try
+        continue-on-error: true # Retry after first failure
+        run: npm install -g sfdx-cli
+      - name: SFDX install second try
+        if: ${{ steps.sfdx_first_try.outcome == 'failure' }}
+        id: sfdx_second_try
+        continue-on-error: true # Retry after second failure
+        run: npm install -g sfdx-cli
+      - name: SFDX install third try
+        if: ${{ steps.sfdx_second_try.outcome == 'failure' }}
+        id: sfdx_third_try
+        run: npm install -g sfdx-cli
+
+      # === Make three attempts to install the scanner plugin ===
+      - name: Scanner install first try
+        id: scanner_first_try
+        continue-on-error: true # Retry after first failure
+        run: echo y | sfdx plugins:install @salesforce/sfdx-scanner
```

Comment:
This should not need `echo y` since it is signed.

Reply:
Good point, I'll remove it.